src/FileConventions/Library.fs

```diff
@@ -37,3 +37,13 @@ let MixedLineEndings(fileInfo: FileInfo) =
         |> Seq.length
 
     numberOfLineEndings > 1
+
+let DetectUnpinnedVersionsInGitHubCI(fileInfo: FileInfo) =
+    // ToDo: assert yml file

```

Comment:
@realmarv what is this?

Reply:
I thought maybe we should make sure the input file to this function is a .yml file

Reply:
great thinking; it takes 1 minutes to add an assert that makes sure the file has a .yml extension, let's add it instead of adding the TODO

Reply:
Done